packages/tools/src/utilities/contours/getDeduplicatedVTKPolyDataPoints.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Iterate through polyData from vtkjs and merge any points that are the same
+ * then update merged point references within lines array
+ * @param polyData - vtkPolyData
+ * @param bypass - bypass the duplicate point removal
+ * @returns the updated polyData
+ */
+export function getDeduplicatedVTKPolyDataPoints(polyData, bypass = false) {
```

Comment:
Could you add a unit test for this function?

Reply:
I just renamed that function, I will add todo